opensrp-chw/src/main/res/values-fr/strings.xml

```diff
@@ -293,6 +293,7 @@
     <!-- PNC Home visit subtitle text -->
     <string name="pnc_danger_signs_mama">Signes de danger - femme</string>
     <string name="pnc_danger_signs_baby_task">Signes de danger - enfant</string>
+    <string name="pnc_skin_to_skin">KMC skin-to-skin counselling</string>
```

Comment:
For the one we are doing at the moment, is it PNC or CHILD? I know we have this content in both pnc and child

Reply:
@issyzac - We are doing Child but based on the discussion with @cozej4 and maybe Mario, the Child Visits/Content for a child before 28 days is being done at PNC.

Reply:
Got it!